src/App.js

```diff
@@ -1,40 +1,54 @@
-import { BrowserRouter, Route, Routes} from "react-router-dom"
+import { BrowserRouter, Link, Route, Routes} from "react-router-dom"
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Navbar from 'react-bootstrap/Navbar';
 import Nav from 'react-bootstrap/Nav';
+import LinkContainer from 'react-router-bootstrap/LinkContainer';
+import { useState } from 'react';
 
+const langs = [{language:"C", resources:["Harvard's CS50X"]}, {language:"Python 3", resources:["Python Crash Course", "Harvard's CS50X", "Harvard's CS50P", "Code Combat", "The Big Book of Small Python Projects"]}, {language:"HTML/CSS", resources:["Harvard's CS50X", "Harvard's CS50W", "FreeCodeCamp"]}, {language:"JavaScript", resources:["Harvard's CS50W", "Mozilla Developer Network", "Code Combat"]}, {language:"SQL", resources:["Harvard's CS50X"]}, {language:"Django", resources:["Harvard's CS50W"]}, {language:"React", resources:["Harvard's CS50W", "React Documentation"]}];
 
 function NavBar() {
   return (
     <Navbar>
       <Navbar.Brand><img alt="Cute Mushroom" src="brand.png" width="40" height="40" /></Navbar.Brand>
       <Nav>
-        <Nav.Link href="/">Homepage</Nav.Link>
-        <Nav.Link href="/languages">Languages</Nav.Link>
+        <LinkContainer to="/"><Nav.Link>Homepage</Nav.Link></LinkContainer>
+        <LinkContainer to="/languages"><Nav.Link>Languages</Nav.Link></LinkContainer>
+        <LinkContainer to="/about"><Nav.Link>About Me</Nav.Link></LinkContainer>
       </Nav>
     </Navbar>
   );
 }
-function LangItem({ lang }) {
+function LangItem({ lang , active, onShow }) {
   let language = lang.language;
   let resources = lang.resources.join(", ");
   return (
-    <tr>
+    <tr onClick={onShow}>
       <td>
       <div className="table-lang"><b>{language}</b></div>
-      <div className="table-resource">Resources:  {resources}</div>
+      {active == language ? (
+        <div className="table-resource">Resources:  {resources}</div>
+      ) : (
+        <div></div>
+      )}
       </td>
     </tr>
   );
 }
 
 function LangList() {
-  const langs = [{language:"C", resources:["Harvard's CS50X"]}, {language:"Python 3", resources:["Python Crash Course", "Harvard's CS50X", "Harvard's CS50P", "Code Combat", "The Big Book of Small Python Projects"]}, {language:"HTML/CSS", resources:["Harvard's CS50X", "Harvard's CS50W", "FreeCodeCamp"]}, {language:"JavaScript", resources:["Harvard's CS50W", "Mozilla Developer Network", "Code Combat"]}, {language:"SQL", resources:["Harvard's CS50X"]}, {language:"Django", resources:["Harvard's CS50W"]}, {language:"React", resources:["Harvard's CS50W", "React Documentation"]}];
+  const [active, setActive] = useState("None");
   const rows = [];
```

Comment:
instead of making an array `rows` and pushing individual rows to it, a more typical way to do things would be to use `Array.map` wherever the rows var is being read. 

In this case,
```
return (
    <table className="langtable">
      <thead>
        <th>Languages</th>
      </thead>
      <tbody>{langs.map(lang => <LangItem lang={lang} />)</tbody>
    </table>
    
  );
  ```